OracleWebLogic/dockerfiles/12.2.1/Dockerfile.infrastructure

```diff
@@ -53,8 +53,7 @@ ENV ORACLE_HOME=/u01/oracle \
 # ------------------------------------------------------------
 RUN mkdir -p /u01/oracle && \
     chmod a+xr /u01 && \
-    useradd -b /u01 -m -s /bin/bash oracle && \
-    echo oracle:oracle | chpasswd && \
+    useradd -b /u01 -M -s /bin/bash oracle && \
```

Comment:
If you're using `-M` to not create a home directory, I would set `-d /u01/oracle` so that the home directory is set to `/u01/oracle` for the `oracle` user.